server/types/listing.ts

```diff
@@ -7,6 +7,7 @@ export type ListingId = CouchUuid
 
 export interface Listing extends CouchDoc {
   _id: ListingId
+  type: string
```

Comment:
```suggestion
  type: keyof typeof entityTypesByListingType
```
Having a more narrow types will help with things such as detection of the missing/invalid per-type i18n keys  client side

Reply:
done in ed0a6b78